src/components/App/index.js

```diff
@@ -1,19 +1,25 @@
 import React from "react";
-
 import Routing from "../../routing";
 import Header from "../Header";
+import Footer from "../Footer";
+// import "./styles/styles.css";
+import { ThemeProvider } from "@mui/material/styles";
+import theme from "../../theme/theme";
 
 import "./styles.scss";
 
 function App() {
   return (
-    <div className="layout">
-      <Header />
-      <div className="content">
-        {" "}
-        <Routing />{" "}
+    <ThemeProvider theme={theme}>
+      <div className="layout">
+        <Header />
+        <div className="content">
+          {" "}
+          <Routing />{" "}
+        </div>
+        <Footer />
```

Comment:
(Refer to previous comment to see where `showFooter` comes in.)
```suggestion
        <Footer style = {{display: showFooter ? "inherit" : "none"}}/>
```